config/config.go

```diff
@@ -154,6 +154,7 @@ type TracingConfig struct {
 type IstioConfig struct {
 	ComponentStatuses        ComponentStatuses `yaml:"component_status,omitempty"`
 	ConfigMapName            string            `yaml:"config_map_name,omitempty"`
+	EnvoyAdminLocalPort      int               `yaml:"envoy_admin_local_port,omitempty"`
```

Comment:
Small question about the configuration & the additional rights that are required: what if the users don't want to grant this new permissions to Kiali, because they don't want Kiali to sneak in envoy internals?
Do they just have to not grant those permissions, and that's it; an error will be thrown when they click on "show envoy details", but nothing else unexpected happens?

Reply:
>  what if the users don't want to grant this new permissions to Kiali, because they don't want Kiali to sneak in envoy internals?

I don't think this is optional for users, granularity of permissions should be the namespace.

If they want to manually ungrant permissions not this or other feature, they should expect an error, but I don't think we should support a Kiali "à la carte" with this permission removed.

For example, I see this at same level like workload or pod annotations as we have today.

But I guess it would help if the error message is clear, because I see more probably some problem if user didn't upgrade correctly or something similar.